src/themes/admin_default/assets/fossbilling.js

```diff
@@ -1,10 +1,9 @@
 import './scss/fossbilling.scss';
 
 import './js/sprite';
-import './js/jquery.min';
+import $ from 'jquery';
+import 'jquery.browser';
```

Comment:
This has been depreciated and we shouldn't really be adding it in. Probably best to refactor wherever it is required to avoid using this.

Reply:
jQuery Alert is an old plugin based on jQuery v1. `jquery.browser` is added as a temporary fix until we replace jQuery Alert.

Reply:
@evrifaessa is working on a replacement at PR https://github.com/FOSSBilling/FOSSBilling/pull/903.

Reply:
I think I won't be able to remove jQuery Alert completely in that PR. I was planning for a gradual migration, then removing it when it's no longer used.

Reply:
Just checked again. There is only one place left where jAlert is used. jConfirm is cleared. jPrompt is still used in 2 places. I haven't really implemented inputs, forms or anything, so replacing jPrompt will have to wait.

I might not have time to work on replacing jPrompt, so I'd just love to have my PR merged as is before it ends up in conflict hell.

Reply:
Noted - hopefully we can remove this once the underlying requirement is gone.